extensions/legit/legit-common.lisp

```diff
@@ -24,21 +24,32 @@
   ;; Doing this helps avoiding tight coupling between the porcelain package and Lem.
   `(call-with-porcelain-error (lambda () ,@body)))
 
+(defvar *vcs-existence-order*
```

Comment:
this var and this function were in the porcelain package, why move them here?

Reply:
Ah, so this was to break a cyclic dependency.

As you note, the var/function were originally in lem/porcelain. But with my changes, these functions are moved to their own packages. These packages themselves depended on `lem/porcelain`, so I couldn't keep these two things in `lem/porcelain`.

`legit-common` was the only spot these functions are used anyways, so moving them was a really easy solution.

I guess an alternative would be to just export `*vcs-existence-order*` from `lem/porcelain`, and have the `git/hg/fossil` packages `push` themselves onto it: but I'm going to assume this is fine for now.